content/en/universal_service_monitoring/_index.md

```diff
@@ -272,6 +273,21 @@ DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED=true
 ```
 
 {{% /tab %}}
+{{% tab "Windows" %}}
+
+**For services running on IIS:**
```

Comment:
Should we call out what a customer should do if they're not using IIS? Opening a support ticket would be a good call-to-action

Reply:
If we do, we should instead put that call to action in the "Supported versions" section -- if you don't see your platform listed, contact support. And you'd need to update the internal confluence page on what to do when the customer calls support from this CTA. What do you think?

Reply:
Sounds good to me!